xml_converter/src/attribute/color.hpp

```diff
@@ -7,18 +7,25 @@
 
 class XMLError;
 namespace waypoint {
-class Color;
+class RGBAColor;
 }
 
 class Color {
  public:
-    std::string hex;
+    float red;
+    float green;
+    float blue;
+    float alpha;
+    int r;
+    int g;
+    int b;
+    int a;
```

Comment:
I have reduced it to one set of floats and added helper functions to switch between int and float to maintain one set of values.

Reply:
Alpha is written as a float in the xml files. RGB are not so don't have a convention. Do you think it makes more sense to store these three as int instead of float? Is there any difference?